orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -419,6 +427,14 @@ public void resolveStrategyParams() {
     return Optional.empty();
   }
 
+  /**
+   * Returns the top-most stage start time.
+   */
+  @JsonIgnore public Long getTopLevelStartTime() {
```

Comment:
Do we need this as a dedicated method? Looks like it's only called in one spot and could just as well be ...

`val startTime = stage.getTopLevelStage().getStartTime().toInstant()`

?

Reply:
Changing, I agree.